Sources/Swagger.swift

```diff
@@ -9,50 +9,17 @@ public struct Swagger {
     /// Provides metadata about the API. The metadata can be used by the clients if needed.
```

Comment:
### Swagger
As per new specification of [OAS Object](https://swagger.io/specification/#oasObject):
- Version field 'swagger' replaced to 'openapi', with versioning (3.0.*)
- Reusable components moved to 'components' field ([Components Object](https://swagger.io/specification/#componentsObject))
- 'host' and 'basePath' replaced with 'servers' (array of [Server Object](https://swagger.io/specification/#serverObject))

#### Merge proposal(s):
Possibly, we can leave the swagger and openapi CodingKeys in top level Swagger structure, but depending on what key exists - create and build two separate implementation of e.g. OpenAPI2 and OpenAPI3 objects containing all other declaration.